packages/storybook/stories/NonReact/NonReactComposites.stories.mdx

```diff
@@ -0,0 +1,71 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+import { Canvas, Title } from '@storybook/addon-docs/blocks';
+
+<Meta id="use-composite-in-non-react-environment" title="Composites/Use Composite In Non React Environment" />
+
+# Use composites under non-react environment
+
+Although the whole composites UI part is built on top of React, you are still able to embed chat/calling composites into your html page with a simple `<script>` tag and several lines of code like:
```

Comment:
If you are not using React, you can still embed a calling/chat composite into your html page with a simple `<script>` tag and several lines of code like: